File: user-api/src/integTest/java/com/gitlab/maly7/relational/ApplicationStartupTests.java
package com.gitlab.maly7.relational;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;
import org.springframework.test.context.junit4.SpringRunner;

@RunWith(SpringRunner.class)
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
@ActiveProfiles(profiles = {"integration"})
public class ApplicationStartupTests {

    @Test
    public void contextLoads() {

    }
}

File: conversation-api/src/integTest/kotlin/io/echoseven/kryption/ApplicationStartupTests.kt
package io.echoseven.kryption

import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.junit4.SpringRunner

@RunWith(SpringRunner::class)
@ActiveProfiles("integration")
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
class ApplicationStartupTests {

    @Test
    fun contextLoads() {

    }
}